### ISSUE TITLE
Todo app example doesn't add new tasks

### ISSUE THREAD

**Issue by marimeireles** [pyscript/pyscript#485]
### Checklist

- [X] I added a descriptive title
- [X] I searched for other issues and couldn't find a solution or duplication
- [X] I already searched in Google and didn't find any good information or help

### What happened?

When you click the new task button, it doesn't add a new task, instead it throws the following error:

```
Uncaught Error: This borrowed proxy was automatically destroyed at the end of a function call. Try using create_proxy or create_once_callable.
The object was of type "function" and had repr "<function on_click_new_task_btn at 0xd7e878>"
```

Reproducible in both chrome and ff.

**Comment by marimeireles** [comment]
I think this is fixed but not released.
I think we should merge a few of the PRs that are ready and then should release a new version soon. What do you think @pww217 ?

**Comment by aqemery** [comment]
> since that alpha is sort of frozen in time at the moment and I don't think we plan to update it.

Is there an easy way to point to latest? I think it would be good to have that document on the main pyscript site. I would rather have the latest and have it break occasionally. This would be fine with the alpha warnings in place.

**Comment by pww217** [comment]
@aqemery this explains the versioning and reasons for it: https://anaconda.cloud/announcing-pyscript-versioning

TLDR: If you want to track the main branch of this repo, it's `unstable`. `latest` is latest Alpha, and then there are named versions.

The original `/alpha/` is basically legacy support. So I think we should just point it to `/latest`. @fpliger any input?

**Comment by aqemery** [comment]
@pww217 awesome, thanks

**Comment by fpliger** [comment]
I think this is a composite issue... the issue on `py-button` and `py-input` themselves should have been resolved but some of the issues bubbling up in examples are related to some of the recent changes to examples and [lack of] tests. We are planning a release this month and I think a lot of the related issues will be fixed (and more visible)

**Comment by marimeireles** [comment]
Was also fixed.
